jscomp/syntax/tests/printer/expr/jsx.res

```diff
@@ -6,13 +6,17 @@ let x = <Foo.bar className="container" />
 let x = <Foo.baz
  className="multiline"
 />
+let x = <\"custom-tag" className="container" />
```

Comment:
Side note, but given that this looks like a close tag (but is not), are there other issues this could cause? How can we test those?

Reply:
yes readability suffers a bit with this, maybe we could change the parser to make it accept exotic names inside double quotes without escaping them with a `\`, like polymorphic variants, but this would be more work for something that is definitely a corner case.

Reply:
Probably will be easier to make `<my-custom-tag className="hello" />` compile in the case